### PR TITLE
feat(cli): create `prisma migrate new` command

### PR DESCRIPTION
**Pull request by gutyerrez**
This PR add `prisma migrate new` command.

It will give a another way to create migrations not directly related to schema. Giving a new way on changing something on database and storing that on the code repository.

At this moment i've not found an issue related to this PR

### CODE REVIEW

**Comment by gutyerrez** [comment]
Note: need to fix errors because another changes was made on migrate. i'll fix it

File: packages/migrate/src/commands/MigrateNew.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        migrationName: migrationName ?? '',
+        draft: true,
+        // Passing an empty prismaSchema will create an empty migration.
+        prismaSchema: '',
+      })
+
+      debug({ createMigrationResult })
+
+      process.stdout.write(
+        `\nThe following migration have been created:\n\n${printMigrationId(
```
Comment (janpio) [inline packages/migrate/src/commands/MigrateNew.ts:124]:
```suggestion
        `\nThe following migration has been created:\n\n${printMigrationId(
```

File: packages/migrate/src/__tests__/MigrateNew.test.ts [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+afterEach(() => {
+  captureStdout.clearCaptureText()
+})
+
+afterAll(() => {
+  captureStdout.stopCapture()
+})
+
+describe('common', () => {
+  it('invalid schema', async () => {
```
Comment (janpio) [inline packages/migrate/src/__tests__/MigrateNew.test.ts:33]: Should this actually happen if you just want to create an _empty_ SQL migration file independent from your Prisma Schema?

File: packages/migrate/src/__tests__/MigrateNew.test.ts [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+  it('help flag', async () => {
+    const commandInstance = MigrateNew.new()
+    const spy = jest.spyOn(commandInstance, 'help').mockImplementation(() => 'Help Me')
+
+    await commandInstance.parse(['--help'])
+    expect(spy).toHaveBeenCalledTimes(1)
+    spy.mockRestore()
+  })
+
+  it('should fail if no schema file', async () => {
```
Comment (janpio) [inline packages/migrate/src/__tests__/MigrateNew.test.ts:111]: Is the schema file really required?

**Pull request closed without merging**